.github/workflows/release.yml

```
@@ -0,0 +1,150 @@
+      - name: Generate PyInstaller version-info resource
+        shell: pwsh
+        run: |
+          $version = (Get-Content plexmuxy/VERSION -Encoding utf8).Trim()
+          if ($version -match '^(\d+)\.(\d+)\.(\d+)') {
+            $filevers = "$([int]$Matches[1]), $([int]$Matches[2]), $([int]$Matches[3]), 0"
+            $prodvers = $filevers
+          } else { throw "Cannot parse version '$version'" }
+          $info = @"
+# UTF-8
+#
+VSVersionInfo(
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)